Dockerfile

```
@@ -1,4 +1,6 @@
-FROM php:7.4-cli-alpine3.16
+ARG PHP_VERSION
```

```suggestion
ARG PHP_VERSION
ARG XDEBUG_VERSION=3.1.2
```

And then `xdebug-${XDEBUG_VERSION}` below. In #6833 I used 3.2.0 where support for PHP 7.4 was dropped and we need differentiate XDebug version between builds.

Maybe 3.2.0 can be used as default and then 3.1.2 explicitly provided as arg for 7.4 build.